_pages/dev/components/banner-component.md

```diff
@@ -2,62 +2,62 @@
 
-The different types have been created for historical reasons, there's not much of a point to distinquish them anymore; there's only component implementation in Spartacus for banners. Sparatacus always renders banners using adaptive and responsive features to render the most optimal image for the given device.
+The different banner component types have been created for historical reasons, but there is not much to distinguish them anymore: in Spartacus, there is only component implementation for banners. Spartacus always renders banners using adaptive and responsive features to render the optimal image for the given device.
 
-The CMS Banner component is mapped to the Spartacus implementation as follows:
+The CMS banner component is mapped to the Spartacus implementation as follows:
 
 ```typescript
 <CmsConfig>{
-    SimpleResponsiveBannerComponent: {
-        component: BannerComponent
-    },
-    BannerComponent: {
-        component: BannerComponent
-    },
-    SimpleBannerComponent: {
-        component: BannerComponent
-    }
+  SimpleResponsiveBannerComponent: {
+    component: BannerComponent,
+  },
+  BannerComponent: {
+    component: BannerComponent,
+  },
+  SimpleBannerComponent: {
+    component: BannerComponent,
+  },
 };
 ```
 
-You can use this component mapping to configure an alternative banner implementation. You might need to map all CMS banner component types in case multiple types are used in your project.
+You can use this component mapping to configure an alternative banner implementation. For example, you might need to map all CMS banner component types if multiple types are used in your project.
 
-## Adaptive vs responsive images
+## Adaptive and Responsive Images
 
-The banner component supports both _responsive_ images as well as _adaptive_ images. Responsive images are simply driven by the CSS rules, which allow to reuse the same images cross different devices and screen sizes. Adaptive images can be used simultaneouly, and are driven by the images provided by the backend. The backend provides different medias by media formats. The `cx-media` components adds the different formats in a so-called `srcset`, so that the browser decides when to use which image.
+The banner component supports both responsive images and adaptive images. Responsive images are simply driven by CSS rules, which allow you to reuse the same images across different devices and screen sizes. Adaptive images can be used simultaneously, and are driven by the images provided by the back end. The back end provides different media items according to the media format. The `cx-media` component adds the different formats in a `srcset` so that the browser can decide when to use which image. Media items are rendered in the `cx-media` component. For more information, see [Media Component]({{ site.baseurl }}{% link _pages/dev/components/shared-components/media-component.md %}).
 
 ## Component Data
 
 The banner component renders the following properties:
```

Comment:
Can we add: 

The banner component renders the following properties. Please note that some of those properties might not be available depending on the banner component type you receive from the backend.

Reply:
👍